crates/biome_js_analyze/src/lint/a11y/no_blank_target.rs

```diff
@@ -177,3 +195,17 @@ impl Rule for NoBlankTarget {
         ))
     }
 }
+
+#[derive(Clone, Debug, Default, Deserialize, Deserializable, Eq, PartialEq, Serialize)]
+#[cfg_attr(feature = "schema", derive(schemars::JsonSchema))]
+#[serde(rename_all = "camelCase", deny_unknown_fields)]
+pub struct AllowDomainOptions {
+    #[serde(skip_serializing_if = "Vec::is_empty")]
+    pub allow_domains: Vec<String>,
```

Comment:
This option should be documented using Rust doc comments `///`. The documentation will be part of the configuration schema, used by the users